### DISCUSSION TITLE
Getting console name from App version

### DISCUSSION THREAD

**Discussion by delislej** [darthcloud/BlueRetro#284] [General]
Hi there, i was hoping to attempt automated filtering (that you can switch to all) by identifying the console from the App version.

I was wondering if the naming scheme would stay the same, like "v#.## console_spiffs"

For example: App version: v1.5 n64_spiffs 

I could detect n64 from that using regex.

On load the presets section of the app would get the console from the app version function and pre-filter the list. you could click an advanced box to get "all" of the presets.

Thanks!

**Comment by darthcloud** [comment]
That changed today, but the console name wont change, so that part should be safe to regex.
